packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/translations/Messages_es_ES.json

```diff
@@ -32,7 +32,7 @@
   "hosted_email_detail_exchange_standard_1g": "Exchange 1 GB",
   "hosted_email_detail_exchange_basic_1g": "Exchange 1 GB",
   "hosted_email_detail_exchange_standard": "Exchange 25 GB",
-  "hosted_email_detail_hostedemail_individual": "Hosted 25 GB",
+  "hosted_email_detail_individual": "Hosted 25 GB",
```

Comment:
`hosted_email_detail_hostedemail_individual` removed, `hosted_email_detail_individual` moved in file.